components/index.css

```diff
@@ -32,6 +32,7 @@
 @import "./button-link/dist/index.css";
 @import "./button/dist/index.css";
 @import "./calendar/dist/index.css";
+@import "./card/dist/index.css";
```

Comment:
Deze file wordt niet meer gebruikt denk ik, waarschijnlijk kan 'ie weg. Ik heb vandaag ook al de `@utrecht/components` package deprecated gemaakt.

De file die wél nog geupdate moet worden is deze:

https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-css/src/index.scss